### PR TITLE
[TH2-3893] Update for latest Cradle dev version

### PR DESCRIPTION
**Pull request by OptimumCode**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ plugins {
 
 ext {
     dockerImageVersion = release_version
-    cradleApiVersion = '5.0.0-dev-version-5-3911117271-0064b41-SNAPSHOT'
+    cradleApiVersion = '5.0.0-dev-version-5-4252457586-c1a33ab-SNAPSHOT'
```
Comment (Nikita-Smirnov-Exactpro) [inline build.gradle]: Maybe switch to dev release for dev-version. We can see workflow example in common-j repository

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (OptimumCode) [reply build.gradle]: @Nikita-Smirnov-Exactpro When can we expect a dev release for cradle library?

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle]: We can make cradle API dev release after checking backward compatibility with previous release version.

File: src/main/kotlin/com/exactpro/th2/lwdataprovider/handlers/util/StoredMessageFilterUtil.kt [hunk 2]
```diff
@@ -25,7 +25,7 @@ fun MessageFilterBuilder.modifyFilterBuilderTimestamps(request: SseMessageSearch
         request.startTimestamp?.let { timestampFrom().isGreaterThanOrEqualTo(it) }
         request.endTimestamp?.let { timestampTo().isLessThan(it) }
     } else {
-        request.endTimestamp?.let { timestampFrom().isGreaterThanOrEqualTo(it) }
-        request.startTimestamp?.let { timestampTo().isLessThan(it) }
+        request.endTimestamp?.let { timestampFrom().isGreaterThan(it) }
+        request.startTimestamp?.let { timestampTo().isLessThanOrEqualTo(it) }
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/kotlin/com/exactpro/th2/lwdataprovider/handlers/util/StoredMessageFilterUtil.kt:29]: Has README contain information about this feature?

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

File: src/main/kotlin/com/exactpro/th2/lwdataprovider/handlers/util/StoredMessageFilterUtil.kt (hunk 2 above)
Comment (OptimumCode) [reply src/main/kotlin/com/exactpro/th2/lwdataprovider/handlers/util/StoredMessageFilterUtil.kt:29]: What feature? This exists for a long time. I just make it consistent with direct search

**Pull request merged**